tools/src/commands/SyncBundledNativeModules.ts

```diff
@@ -0,0 +1,236 @@
+import { Command } from '@expo/commander';
+import JsonFile from '@expo/json-file';
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import fetch from 'node-fetch';
+import path from 'path';
+import semver from 'semver';
+
+import { EXPO_DIR, LOCAL_API_HOST, STAGING_API_HOST, PRODUCTION_API_HOST } from '../Constants';
+import logger from '../Logger';
+
+type ActionOptions = {
+  env: string;
+};
+
+type Env = 'local' | 'staging' | 'production';
+type BundledNativeModules = Record<string, string>;
+interface NativeModule {
+  npmPackage: string;
+  versionRange: string;
+}
+type BundledNativeModulesList = NativeModule[];
+interface SyncPayload {
+  nativeModules: BundledNativeModulesList;
+}
+interface GetBundledNativeModulesResult {
+  data: BundledNativeModulesList;
+}
+
+const EXPO_PACKAGE_PATH = path.join(EXPO_DIR, 'packages/expo');
+
+async function main(options: ActionOptions) {
+  logger.info('\nSyncing bundledNativeModules.json with www...');
+
+  const env = resolveEnv(options);
+  await confirmEnvAsync(env);
+  const secret = await resolveSecretAsync();
+
+  const sdkVersion = await resolveTargetSdkVersionAsync();
+  const bundledNativeModules = await readBundledNativeModulesAsync();
+  const syncPayload = prepareSyncPayload(bundledNativeModules);
+
+  const currentBundledNativeModules = await getCurrentBundledNativeModules(env, sdkVersion);
+  await compareAndConfirmAsync(currentBundledNativeModules, syncPayload.nativeModules);
+
+  await syncModulesAsync({ env, secret }, sdkVersion, syncPayload);
+  logger.info(chalk.green(`Successfully synced the modules for SDK ${sdkVersion}!`));
```

Comment:
As the entire log is green, I'd suggest `logger.success`
```suggestion
  logger.success(`Successfully synced the modules for SDK ${sdkVersion}!`);
```